templates/sshd_config.erb

```diff
@@ -14,7 +14,13 @@
 # default value.
 
 #Port 22
+<% if @sshd_config_port.class == Array -%>
+<% @sshd_config_port.each do |p| -%>
+<%= "Port #{p}" %>
+<% end -%>
+<% else -%>
 Port <%= @sshd_config_port %>
+<% end -%>
```

Comment:
Just a side comment:
usually I try to avoid functional logic in templates. In this case you could normalize the validated data to become an array before it reaches the template.